DESCRIPTION

```diff
@@ -35,6 +36,8 @@ Suggests:
     rhdf5,
     rmarkdown,
     magrittr
+Remotes: 
```

Comment:
Actually, @sgibb and I opted some time ago against the use of a `Remote`. During development we will anyway have to have `MsCoreUtils` installed locally and later, in Bioconductor, we don't need it anymore. Having it now might increase the chances to forget removing it later.